interfaces/Portal/src/app/pages/login/login.page.ts

```diff
@@ -46,13 +51,27 @@ export class LoginPage implements OnDestroy {
 
   private msalSubscription: Subscription;
 
+  public ssoUserIsNotFound: boolean;
+  public isPopupBlocked = false;
+
   constructor(
     private authService: AuthService,
     private translate: TranslateService,
     private router: Router,
     private msalService?: MsalService,
   ) {}
 
+  ngOnInit(): void {
```

Comment:
On line 24, after `implements`, `OnInit` should be added. (and imported as well)

Reply:
Done.